front/lib/strapi/strapiCRUD.ts

```diff
@@ -6,6 +6,9 @@ const base_url = 'http://localhost:1337/api/';
 export const strapiGet = async ({ req_url, filters }: StrapiCRUDType) =>
     await axios.get(base_url + req_url + '?' + qs.stringify({ filters }));
 
+export const strapiGetPopulation = async ({ req_url, req }: StrapiCRUDType) =>
```

Comment:
I'd merge it with strapiGet by adding to strapiGet additional parameter like (req_url, filters, population)